### PR TITLE
Added metadata docs for light entities

### PR DESCRIPTION
**Pull request by est77**
Part of issue #745.

### CODE REVIEW

File: src/appleseed/renderer/modeling/light/ilightfactory.cpp [hunk 1]
```diff
@@ -55,7 +56,8 @@ void ILightFactory::add_common_input_metadata(DictionaryArray& metadata)
             .insert("label", "Importance Multiplier")
             .insert("type", "text")
             .insert("use", "optional")
-            .insert("default", "1.0"));
+            .insert("default", "1.0")
+            .insert("help", "TODO: document me"));
```
Comment (est77) [inline src/appleseed/renderer/modeling/light/ilightfactory.cpp]: Missing doc string

File: src/appleseed/renderer/modeling/light/spotlight.cpp [hunk 2]
```diff
@@ -286,7 +291,8 @@ DictionaryArray SpotLightFactory::get_input_metadata() const
             .insert("min_value", "-360.0")
             .insert("max_value", "360.0")
             .insert("use", "optional")
-            .insert("default", "0.0"));
+            .insert("default", "0.0")
+            .insert("help", "TODO: document me"));
```
Comment (dictoon) [inline src/appleseed/renderer/modeling/light/spotlight.cpp]: "Rotate the spot light around its axis; only useful when using the light intensity is textured (gobo)"

File: src/appleseed/renderer/modeling/light/sunlight.cpp [hunk 3]
```diff
@@ -445,7 +445,8 @@ Dictionary SunLightFactory::get_model_metadata() const
     return
         Dictionary()
             .insert("name", Model)
-            .insert("label", "Sun Light");
+            .insert("label", "Sun Light")
+            .insert("help", "Physically based sun light");
```
Comment (dictoon) [inline src/appleseed/renderer/modeling/light/sunlight.cpp]: "Phyically-based sun light"

File: src/appleseed/renderer/modeling/light/ilightfactory.cpp (hunk 1 above)
Comment (dictoon) [inline src/appleseed/renderer/modeling/light/ilightfactory.cpp]: "Adjust the sampling effort for this light with respect to the other lights."

**Pull request merged**